Vectra/vectra_cognito_detect/tests/vectra_info.json

```
@@ -6,42 +6,39 @@
                 "dialect_uuid": "bf8867ee-43b7-444c-9475-a7f43754ab6d"
             }
         },
-        "message":"-: {\"category\": \"INFO\", \"certainty\": 0, \"d_type\": \"si_new_host\", \"d_type_vname\": \"New Host\", \"dd_bytes_rcvd\": null, \"dd_bytes_sent\": null, \"dd_dst_dns\": \"\", \"dd_dst_ip\": \"0.0.0.0\", \"dd_dst_port\": 80, \"dd_proto\": \"\", \"detection_id\": 9999, \"dvchost\": \"255.255.255.1\", \"headend_addr\": \"255.255.255.1\", \"host_ip\": \"10.0.0.1\", \"host_name\": \"plop-99\", \"href\": \"https://255.255.255.1/detections/9999?detail_id=11111\", \"severity\": 0, \"threat\": 0, \"triaged\": false, \"vectra_timestamp\": \"1099999999\", \"version\": \"6.7\"}"
+        "message": "-: {\"category\": \"INFO\", \"certainty\": 0, \"d_type\": \"si_new_host\", \"d_type_vname\": \"New Host\", \"dd_bytes_rcvd\": null, \"dd_bytes_sent\": null, \"dd_dst_dns\": \"\", \"dd_dst_ip\": \"0.0.0.0\", \"dd_dst_port\": 80, \"dd_proto\": \"\", \"detection_id\": 9999, \"dvchost\": \"255.255.255.1\", \"headend_addr\": \"255.255.255.1\", \"host_ip\": \"10.0.0.1\", \"host_name\": \"plop-99\", \"href\": \"https://255.255.255.1/detections/9999?detail_id=11111\", \"severity\": 0, \"threat\": 0, \"triaged\": false, \"vectra_timestamp\": \"1099999999\", \"version\": \"6.7\"}"
     },
     "expected": {
         "event": {
             "type": "INFO",
             "url": "https://255.255.255.1/detections/9999?detail_id=11111"
         },
-        "observer":
-        {
-            "version":"6.7",
-            "name":"255.255.255.1",
-            "ip":"255.255.255.1"
+        "observer": {
+            "version": "6.7",
+            "name": "255.255.255.1",
+            "ip": "255.255.255.1"
```

Shouldn't this be something like this?
```suggestion
            "ip": ["255.255.255.1"]
```

---

right https://www.elastic.co/guide/en/ecs/current/ecs-observer.html#field-observer-ip

---

You are right, but this is not the point of this PR